packages/katello/rubygem-pulp_certguard_client/rubygem-pulp_certguard_client.spec

```diff
@@ -1,68 +1,47 @@
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name pulp_certguard_client
-%global gem_require_name %{gem_name}
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.5.0
+Name: rubygem-%{gem_name}
+Version: 1.5.3
 Release: 1%{?dist}
-Summary: Client bindings for pulp certguard plugin
-Group: Development/Languages
-License: GPL-2.0+
+Summary: Pulp 3 API Ruby Gem
+License: GPLv2+
 URL: https://github.com/pulp/pulp-certguard
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby >= 1.9
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(faraday) >= 0.14.0
-Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1.0
-Requires: %{?scl_prefix_ruby}rubygem(json) >= 2.1
-Requires: %{?scl_prefix_ruby}rubygem(json) < 3
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby >= 1.9
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: ruby >= 1.9
+BuildRequires: ruby >= 1.9
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
 %description
-Client bindings for pulp certguard plugin
+Fetch, Upload, Organize, and Distribute Software Packages.
+
 
 %package doc
-Summary: Documentation for %{pkg_name}
-Group: Documentation
-Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+Summary: Documentation for %{name}
+Requires: %{name} = %{version}-%{release}
 BuildArch: noarch
 
 %description doc
-Documentation for %{pkg_name}.
+Documentation for %{name}.
 
 %prep
-%{?scl:scl enable %{scl} - << \EOF}
-gem unpack %{SOURCE0}
-%{?scl:EOF}
-
-%setup -q -D -T -n  %{gem_name}-%{version}
+%setup -q -n  %{gem_name}-%{version}
 
-%{?scl:scl enable %{scl} - << \EOF}
-gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
-%{?scl:EOF}
+# https://github.com/pulp/pulp-openapi-generator/pull/73
```

Comment:
I saw this was merged. Will the next version then remove the need for these two lines below?

Reply:
Correct, but that requires pushing new gems and that's up to the individual plugin owners. I don't want to wait for that to fix repoclosure hence this workaround.

Reply:
In addition to that: the `gemspec_remove_dep` should fail if the exact dependency isn't found, so when it's redundant the build should fail.